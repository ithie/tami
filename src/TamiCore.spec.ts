import { describe, test, expect } from 'vitest'
import type { DefineComponent } from 'vue'
import { mount } from '@vue/test-utils'
import TamiCore from '@/TamiCore.ce.vue'

describe('TamiCore', () => {
  test('should render', async () => {
    const wrapper = mount(TamiCore as DefineComponent)

    expect(wrapper.element).toMatchSnapshot()
  })
})
