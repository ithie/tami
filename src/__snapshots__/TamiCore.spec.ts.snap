// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TamiCore > should render 1`] = `
<div
  class="border-all"
>
  
  <div
    class="wrapper"
  >
    
    
    <button>
      EN
    </button>
    <button>
      DE
    </button>
    <button>
      CS
    </button>
    <button>
      IT
    </button>
    <button>
      ES
    </button>
    <button>
      FR
    </button>
    
    
  </div>
  <div
    style="height: 15px; padding: 0px; margin: 0px; display: block; width: 100%;"
  />
  <div
    class="tami-border"
  >
    <pre>
      (o o)
    </pre>
  </div>
  
  <pre>
    Hunger:       █████     
  </pre>
  <pre>
    Happiness:    █████     
  </pre>
  <pre>
    Energy:       █████     
  </pre>
  <pre>
    Age:          1
  </pre>
  
  <div
    class="wrapper"
  >
    
    <button>
      Feed
    </button>
    <button>
      Play
    </button>
    <button>
      Sleep
    </button>
    <button>
      Nothing
    </button>
    
  </div>
  
</div>
`;
